nixos/modules/system/boot/tmp.nix

```diff
@@ -24,6 +27,14 @@ with lib;
       '';
     };
 
+    boot.tmpOnTmpfsSize = mkOption {
+      type = types.str;
+      default = "50%";
```

Comment:
should the fact that we follow the upstream definition on % be documented? https://github.com/NixOS/nixpkgs/pull/27189#discussion_r126031521